### PR TITLE
[ASM] - Expander - Standardize enrichment names in asmsystemids grid field (4819 & 5508)

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
- [EXPANDR-4819](https://jira-hq.paloaltonetworks.local/browse/EXPANDR-4819)
- [EXPANDR-5508]([EXPANDR-5508](https://jira-hq.paloaltonetworks.local/browse/EXPANDR-5508)

## Description
Updating the `type` field values where `asmsystemids` is set throughout playbooks to use standardized values.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 id: Cortex ASM - Qualys Enrichment
 version: -1
 name: Cortex ASM - Qualys Enrichment
-description: Given the IP address this playbook enriches information from Qualys assets.
+escription: Given the IP address this playbook enriches information from Qualys assets.
```
Comment (MichaelYochpaz) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml:4]:
```suggestion
description: Given the IP address this playbook enriches information from Qualys assets.
```

Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml:4]: that should fix this build error: https://app.circleci.com/pipelines/github/demisto/content/295010/workflows/01496a17-45a3-4eb3-881c-1b92afae8b2f/jobs/716722?invite=true#step-119-164

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml [hunk 2]
```diff
@@ -1128,12 +1128,12 @@ tasks:
       '#none#':
       - "20"
     scriptarguments:
+      append:
+        simple: "true"
       key:
         simple: asm_enrichment_flag_aws
       value:
         simple: "false"
-      append:
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml:1135]: fixed from before?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml [hunk 3]
```diff
@@ -1576,8 +1576,51 @@ tasks:
...
+    isautoswitchedtoquietmode: false
+  "56":
+    id: "56"
+    taskid: 85485dba-f2c7-42c2-864e-b695a9ee0551
+    type: regular
+    task:
+      id: 85485dba-f2c7-42c2-864e-b695a9ee0551
+      version: -1
+      name: Set system IDs grid field (type)
+      description: Sets the Azure Compute instance ID to the grid field for the ASM system IDs object.
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml:1598]:
```suggestion
      description: Sets the Compute instance ID to the grid field for the ASM system IDs object.
```
remove Azure reference

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml [hunk 4]
```diff
@@ -555,175 +555,8 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 710,
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml:558]: Need to make changes from https://github.com/demisto/content/pull/28908 are reflected here

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Rapid7_Enrichment.yml [hunk 5]
```diff
@@ -853,12 +853,12 @@ tasks:
...
     id: "27"
-    taskid: ab2e86ed-b0b3-49a3-85d0-a4ee5db2762d
+    taskid: 1ab733c9-9298-4e95-8ad5-0aaa4b8aba6b
     type: regular
     task:
-      id: ab2e86ed-b0b3-49a3-85d0-a4ee5db2762d
+      id: 1ab733c9-9298-4e95-8ad5-0aaa4b8aba6b
       version: -1
-      name: Set system IDs grid field (Asset ID)
+      name: Set system IDs grid field (asset ID)
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Rapid7_Enrichment.yml:861]: General comment: anywhere were task names are changes, we should update the screenshot of the playbook (light mode), please

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
@BigEasyJ , still looking through, here are a few things

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 6]
```diff
@@ -832,7 +834,7 @@ tasks:
       brand: Builtin
     nexttasks:
       '#none#':
-      - "136"
+      - "110"
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:837]: change back to the way it was.  Changes in https://github.com/demisto/content/pull/28908

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 7]
```diff
@@ -2093,7 +2095,7 @@ tasks:
       brand: ""
     nexttasks:
       '#none#':
-      - "110"
+      - "81"
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:2098]: change back to the way it was.  Changes in https://github.com/demisto/content/pull/28908

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 8]
```diff
@@ -1019,7 +1021,7 @@ tasks:
       brand: ""
     nexttasks:
       '#none#':
-      - "81"
+      - "136"
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:1024]: change back to the way it was.  Changes in https://github.com/demisto/content/pull/28908

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
Parent playbook changes in https://github.com/demisto/content/pull/28908 that didn't come over

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml (hunk 4 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml]: resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Rapid7_Enrichment.yml (hunk 5 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Rapid7_Enrichment.yml:861]: resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml (hunk 6 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml]: resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml (hunk 8 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml]: resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml [hunk 9]
```diff
@@ -1622,6 +1712,6 @@ inputs:
   description: Attack Surface Management Rule ID.
   playbookInputQuery:
 outputs: []
-fromversion: 6.5.0
+fromversion: 6.9.0
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml:1715]:
```suggestion
fromversion: 6.5.0
```
fix validation error

**Review by johnnywilkes — CHANGES_REQUESTED** [review]

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml (hunk 3 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml]: resolved in https://github.com/demisto/content/pull/29098/commits/7faeccb53ba3109b146c85d3643ad539a268dcd0

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml (hunk 9 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml]: resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml (hunk 1 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml]: resolved in https://github.com/demisto/content/pull/29098/commits/7faeccb53ba3109b146c85d3643ad539a268dcd0

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , approved.  You might have to change your tested based on pre-commit tests but not sure: https://github.com/demisto/content/actions/runs/5996766820/job/16261891990?pr=29098

**Comment by johnnywilkes** [comment]
@MichaelYochpaz  / @melamedbn , this is approved form Xpanse side.  Please merge when possible

**Comment by BigEasyJ** [comment]
Made changes suggested by @MichaelYochpaz, however I see precommit failing due to not find the existing file path `./Packs/CortexAttackSurfaceManagement/Playbooks/`.

**Review by MichaelYochpaz — APPROVED** [review]
Looks good. Good job :)

**Pull request merged**